src/org/usfirst/frc/team2706/robot/RobotMap.java

```diff
@@ -22,7 +24,7 @@
     private static final int[] MOTOR_FRONT_LEFT_VALS = {1, 1, 1};
     public static final int MOTOR_FRONT_LEFT = getConstant("MOTOR_FRONT_LEFT");
 
-    private static final boolean[] MOTOR_FRONT_LEFT_INVERTED_VALS = {true, true, false};
+    private static final boolean[] MOTOR_FRONT_LEFT_INVERTED_VALS = {false, false, false};
```

Comment:
I'm assuming these were tested and we knows they are the correct values.     Did we test on the competition Robot yet?